cves/CVE-2015-6826.yml

```
@@ -236,8 +269,8 @@ subsystem:
     If this involves fixing multiple subsystems, you can make this an array.
 
     In 'answer', explain how you arrived at this determination.
-  answer:
-  name:
+  answer: The CVE itself specifies a function in this library.
+  name: avcodec
```

```suggestion
  name: [avcodec, avformat]
```
This vulnerability affected more than one module as well (to the point of requiring it to be updated), despite the code residing in avcodec: https://ubuntu.com/security/notices/USN-2944-1

---

The update in `avformat` was to resolve a different CVE that was bundled in the release. https://ubuntu.com/security/CVE-2016-2326